cmd/nerdctl/ps.go

```diff
@@ -216,6 +217,7 @@ func printContainers(ctx context.Context, client *containerd.Client, cmd *cobra.
 			Ports:     formatter.FormatPorts(info.Labels),
 			Status:    cStatus,
 			Runtime:   info.Runtime.Name,
+			Labels:    formatter.FormatLabels(info.Labels),
```

Comment:
I think it would be better if we make these as an argument instead of default behavior such as **nerdctl ps --labels**

If the container has a lot of labels, I think it would be an info disaster in the terminal for user

Reply:
I referenced this doc.
https://github.com/containerd/nerdctl/blob/b5cdd8aa43c6b40502fc740a14682873574470f9/README.md?plain=1#L780
So, I didn't edit table and wide options.
```bash
nerdctl ps -a
CONTAINER ID    IMAGE                                   COMMAND     CREATED        STATUS                    PORTS    NAMES
a03dccba94d1    docker.io/library/hello-world:latest    "/hello"    2 hours ago    Exited (0) 2 hours ago             hello-world-a03dc
bf6dda95ce71    docker.io/library/hello-world:latest    "/hello"    2 hours ago    Exited (0) 2 hours ago             hello-world-bf6dd
c5b8555bad9c    docker.io/library/hello-world:latest    "/hello"    2 hours ago    Exited (0) 2 hours ago             hello-world-c5b85
```
I'm worried that too many options will be created.

My suggestion is
- without any option, display all
- provide an interface to edit it.

Reply:
SGTM